gpflow/config/__config__.py

```diff
@@ -47,13 +68,17 @@ class Config:
         float: Float data type, float32 or float64
         jitter: Jitter value. Mainly used for for making badly conditioned matrices more stable.
             Default value is `1e-6`.
+        positive_bijector: Method for positive bijector, either "softplus" or "exp".
+            Default is "softplus".
         positive_minimum: Lower level for the positive transformation.
         summary_fmt: Summary format for module printing.
     """
 
     int: type = field(default_factory=lambda: default(_Values.INT))
     float: type = field(default_factory=lambda: default(_Values.FLOAT))
     jitter: float = field(default_factory=lambda: default(_Values.JITTER))
```

Comment:
I don't think this will ensure that the environment variables are converted to type objects / float values (and same for `positive_minimum` below)! again -not fault of this PR- but thought important to flag this up now that I spotted it

Reply:
Indeed, @jahall I will fix it in separate PR.